requirements.txt

```diff
@@ -4,3 +4,4 @@ boto3==1.7.35
 simplejson==3.13.2
 six==1.11.0
 future==0.16.0
+Flask
```

Comment:
It's a Dev requirement, not a prod one (whoever not using flask won't need that package)